server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -426,6 +429,14 @@ public int queryTimeoutSeconds() {
         return queryTimeoutSeconds;
     }
 
+    @JsonIgnore
+    public String getInternalSessionInitScript() {
```

Comment:
how about this?

`ConnectionAdapter` gets different implementation in different env, we may provide a special implementation and we can add `set ob_read_consistency='WEAK'` to session init script.

Reply:
`ConnectionAdapter`'s design is a kind of wield btw. I think `ConnectionAdapter` is not oriented to `Environment` but to `Connection` itself. What I expect if that this interface would have implementations of different dialect types, but the current implementation of `ConnectionAdapter` seems to be oriented to deploy environments.

Reply:
I created a new implementation of `ConnectionInitializer` called `BackupInstanceInitializer` and moved the script into it.